lib/Track.lua

```diff
@@ -274,10 +274,18 @@ function Track:get_shadow_attribute(attribute)
   elseif attribute == "channel" then return track:get_channel()
   elseif attribute == "device"  then return track:get_device()
   elseif attribute == "pair"    then return track:get_pair()
-  elseif attribute == "jf"      then return track:is_jf()
+  elseif attribute == "is_jf"   then return track:get_is_jf()
   end
 end
 
+function Track:get_shadowable_attribute(attribute)
```

Comment:
this is good. i'll need to test but i think this will help with some of the issues.

Reply:
Sounds good!